commit 18aff2297ce844362f28ea5317c289ba154bd33d
Author: Lukas Tribus <lukas@ltri.eu>
Date:   Mon Oct 1 02:00:16 2018 +0200

    DOC: clarify force-private-cache is an option
    
    "boolean" may confuse users into thinking they need to provide
    additional arguments, like false or true. This is a simple option
    like many others, so lets not confuse the users with internals.
    
    Also fixes an additional typo.
    
    Should be backported to 1.8 and 1.7.
    
    (cherry picked from commit 2793578eaf934bbf28f742a35f3a1ae656280324)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/doc/configuration.txt b/doc/configuration.txt
index c69033b1..580194ec 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -1651,7 +1651,7 @@ tune.ssl.cachesize <number>
   this value to 0 disables the SSL session cache.
 
 tune.ssl.force-private-cache
-  This boolean disables SSL session cache sharing between all processes. It
+  This option disables SSL session cache sharing between all processes. It
   should normally not be used since it will force many renegotiations due to
   clients hitting a random process. But it may be required on some operating
   systems where none of the SSL cache synchronization method may be used. In
@@ -6535,7 +6535,7 @@ option smtpchk <hello> <domain>
                                  yes   |    no    |   yes  |   yes
   Arguments :
     <hello>   is an optional argument. It is the "hello" command to use. It can
-              be either "HELO" (for SMTP) or "EHLO" (for ESTMP). All other
+              be either "HELO" (for SMTP) or "EHLO" (for ESMTP). All other
               values will be turned into the default command ("HELO").
 
     <domain>  is the domain name to present to the server. It may only be
